### ISSUE TITLE
Question > Can I prevent direct outside access to UUID.cfargotunnel.com? 

### ISSUE THREAD

**Issue by amitkoth** [cloudflare/cloudflared#841]
Might be a n00by question, sorry. 

I know that I can map a CNAME between my Cloudflare tunnel and Cloudflare load balancer. 

However if someone accidentally or otherwise,  discovers the *.cfargotunnel.com URL - will they be able to reach it from the outside web? 

I want all outside web requests to pass through my subdomain.tallyfy.com _only_ so that there's  no other way in. How would I achieve this?

**Comment by sudarshan-reddy** [comment]
> However if someone accidentally or otherwise, discovers the *.cfargotunnel.com URL - will they be able to reach it from the outside web?


They will not be able to reach this URL: *.cfargotunnel.com. You could do this experiment yourself by curling this endpoint as well.

**Comment by amitkoth** [comment]
I realized that later too, apologies for such a basic question and thanks @sudarshan-reddy

**Comment by sudarshan-reddy** [comment]
Not at all! It means you are thinking through the flow! Keep asking! ❤️

**Comment by amitkoth** [comment]
@sudarshan-reddy FYI we've decided on an architecture where there's a pool on Cloudflare that's tied to a tunnel / via CF load balancer, and the _fallback_ pool is our ALB URL (directly to ALB itself) in case Cloudflare's tunnel has problems. 

Here's unrelated questions that are leaving me uncertain and unable to clarify the benefits in my head - and possibly blog post ideas for you in future:

1. **Performance** of tunnel vs. AWS Global Accelerator - I posted about it [here](https://community.cloudflare.com/t/cloudflare-tunnel-and-argo-vs-aws-global-accelerator/377550). If we go all-in on Cloudflare, a key attraction is enabling Argo Smart Routing on a different domain just for our API requests - because it [doesn't support](https://community.cloudflare.com/t/more-selective-argo-and-argo-tunnel-options/144717) partial paths yet - so that the entire trip from outside edge through Cloudflare and into our AWS VM is highly optimized. Never really "speed of light" but as close as can be. Comments on that would really strengthen the case for both tunnels and Argo Smart Routing.
2. **Stickiness and efficiency** for how Cloudflare Workers handle repeat requests to the tunnel from the same edge node - since we already intercept every API request via a CF worker > into your internal network > and into our tunnel. For each request from a specific outside edge worker, we'd like a long-lived and highly efficient network path between that edge node and our tunnel (into our AWS VM) - I presume there's optimizations that are involved beyond HTTP/3, etc? 

There's been a lot of focus on the security aspects of tunnels, which I like - since obviously zero inbound access is beneficial, but the _performance_ benefits of having a tunnel between internal servers and the outside edge are equally crucial selling points.
